src/controller.rs

```diff
@@ -416,10 +416,33 @@ fn dispatch_fifo_request(
         "kakoune/breadcrumbs" => Box::new(BreadcrumbsParams {
             position_line: state.next()?,
         }),
+        "kakoune/did-change-option" => {
```

Comment:
(doesn't matter here but in general it's better to put such sorting changes in an earlier commit)